Foundation/Summit/2024/program.html

```diff
@@ -0,0 +1,373 @@
+---
+#    Copyright 2024 seL4 Project a Series of LF Projects, LLC.
+#    SPDX-License-Identifier: CC-BY-SA-4.0
+title: Summit program
+---
+
+<h1>
+  seL4 Summit 2024
+</h1>
+<h4>
+  Sydney, Australia
+</h4>
+<h4>
+  15-17 October 2024
+</h4>
+
+<h3>
+   Program
+</h3>
+<p>
+  All times are local time in Sydney, Australia (GMT+11).
+  <!--br>
+  Session chairs in <em>italics</em>.-->
+</p>
+<table id="summit-talks">
+  <tr>
+    <th colspan="3">Day 1&emsp;15 October 2024</th>
+  </tr>
+  <tr>
+    <td>9:00 - 9:10</td>
+    <td>Announcements</td>
+    <td><em>Welcome</em></td>
+  </tr>
+  <tr>
+    <td>9:10 - 10:00</td>
+    <td>Keynote</td>
+    <td>Industrial Scale Proof Engineering for Critical Trustworthy Applications (INSPECTA)<br><span class="summit-abstract-presenter">Darren Cofer</span>,
+    <span class="summit-abstract-affiliation">Collins Aerospace</span>
+    </td>
+  </tr>
+  <tr>
+    <td class="break" colspan="3">Break</td>
+  </tr>
+  <tr>
+    <td>10:30 - 11:00</td>
+    <td>Talk</td>
+    <td>seL4 Verification: Status and Plans<br><span class="summit-abstract-presenter">Gerwin Klein</span>,
```

Comment:
Here, and elsewhere, you should have the full list of authors (as discussed Gerwin won't be presenting both of these); We should keep the full list of authors in any case, and highlight in some way the speaker when/once we know it.

Reply:
I'll put the full list in the program.html. The co-authors' details are also in the speakers spreadsheet in Drive.